### PR TITLE
rm: linux: Include code sample on how to setup DISTRO

### PR DESCRIPTION
**Pull request by angolini**
Signed-off-by: Daiane Angolini <daiane.angolini@foundries.io>

### CODE REVIEW

**Comment by doanac** [comment]
Docs for f46f2b3d4f4c74e84dc91dd5b3e3a8e72d50a08e are browesable at: https://ci.foundries.io/projects/fio-docs/builds/1053/docs/artifacts/html/index.html

File: source/reference-manual/linux/linux-distro.rst [hunk 1]
```diff
@@ -31,6 +33,14 @@ It overrides some configurations from ``lmp`` to generate a friendly system for
...
 * The Linux Kernel binary, along with the required DTB files, are provided as separate files, instead of inside a boot image. This way the binaries can be replaced for testing purposes.
 
+To change the distro on a FoundriesFactory, from the ``ci-scripts`` git repository , update the ``factory-config.yml`` to include the ``DISTRO`` parameter under ``lmp:params``, which result in the following::
+
+    lmp:
+      params:
+        IMAGE: lmp-factory-image
+        DOCKER_COMPOSE_APP: "1"
+        DISTRO: lmp-base
+
```
Comment (angolini) [inline source/reference-manual/linux/linux-distro.rst:43]: I could not find an example on the docs on how to actually change the distro.

I decided to not paste again how to add mfgtool as we have this on other sections, but I would accept suggestions to make this section better....

File: source/reference-manual/linux/linux-distro.rst [hunk 2]
```diff
@@ -31,6 +33,14 @@ It overrides some configurations from ``lmp`` to generate a friendly system for
 
 * The Linux Kernel binary, along with the required DTB files, are provided as separate files, instead of inside a boot image. This way the binaries can be replaced for testing purposes.
 
+To change the distro on a FoundriesFactory, from the ``ci-scripts`` git repository , update the ``factory-config.yml`` to include the ``DISTRO`` parameter under ``lmp:params``, which result in the following::
```
Comment (vanmaegima) [inline source/reference-manual/linux/linux-distro.rst:36]:
```suggestion
To change the distro on a FoundriesFactory, from the ``ci-scripts`` git repository , update the ``factory-config.yml`` to include the ``DISTRO`` parameter under ``lmp:params``, which results in the following:
```

**Review by vanmaegima — APPROVED** [review]
just a minor comment

**Review by munoz0raul — APPROVED** [review]
LGTM

**Comment by doanac** [comment]
Docs for 1dece1cfe8aee0eba398b8f73b3d168cf171a91a are browesable at: https://ci.foundries.io/projects/fio-docs/builds/1055/docs/artifacts/html/index.html

**Comment by doanac** [comment]
Docs for 1dece1cfe8aee0eba398b8f73b3d168cf171a91a are browesable at: https://ci.foundries.io/projects/fio-docs/builds/1056/docs/artifacts/html/index.html

**Review by MrCry0 — APPROVED** [review]

**Review by Tim-Anderson — APPROVED** [review]
LGTM

**Review by mike-scott — APPROVED** [review]
LGTM

**Pull request merged**